enterprise_access/apps/api/serializers.py

```diff
@@ -75,3 +80,21 @@ class Meta:
             'coupon_id',
             'coupon_code'
         ]
+
+
+class SubsidyRequestCustomerConfigurationSerializer(serializers.ModelSerializer):
+    """
+    Serializer for the `SubsidyRequestCustomerConfiguration` model.
+    """
+
+    class Meta:
+        model = SubsidyRequestCustomerConfiguration
+        fields = [
+            'enterprise_customer_uuid',
+            'subsidy_requests_enabled',
+            'subsidy_type'
+        ]
+
+    def update(self, instance, validated_data):
+        validated_data.pop('enterprise_customer_uuid', None)
```

Comment:
[clarification] We remove `enterprise_customer_uuid` from `validated_data` here because we don't want the API user to be able to modify the `enterprise_customer_uuid`, right? I'm wondering if we could utilize `read_only_fields` here ([docs](https://www.django-rest-framework.org/api-guide/serializers/#specifying-read-only-fields))?

Reply:
Yeah that's correct. If we want to put it in read_only_fields, we would need to have a separate serializer for write since enterprise_customer_uuid will be discarded.

Reply:
> Yeah that's correct. If we want to put it in read_only_fields, we would need to have a separate serializer for write since enterprise_customer_uuid will be discarded.

True. Personally, no strong feelings on your approach vs. separate serializer though we do have multiple serializers pattern elsewhere, e.g. `EnterpriseCourseEnrollmentWriteSerializer`, `EnterpriseCustomerUserWriteSerializer`, `EnterpriseCustomerInviteKeyWriteSerializer`.

If kept as is, maybe add a comment to explain why we're popping `enterprise_customer_uuid` as it might not be intuitive at first glance (at least it wasn't for me 😅 ).

Reply:
No strong opinion here either, didn't want to write a separate serializer and add logic to the viewset since the difference is so small 😅 . I can add a comment to clarify.

Reply:
Awesome, works for me!